awx/main/dispatch/worker/callback.py

```diff
@@ -26,89 +30,134 @@ class CallbackBrokerWorker(BaseWorker):
 
     MAX_RETRIES = 2
 
+    def __init__(self):
+        self.buff = {}
+        self.last_write = tz_now()
+
+    def read(self, queue):
+        try:
+            return queue.get(block=True, timeout=.1)
+        except QueueEmpty:
+            return {'event': 'FLUSH'}
+
+    def flush(self, force=False):
+        now = tz_now()
+        if (
+            force or
+            (now - self.last_write).seconds < .1 or
```

Comment:
Make the `.1` a variable somehwere. It would be too easy for a future developer to change the timeout of `.1` above and miss searching for this `.1` usage.